src/Core/Services/NoopImplementations/NoopEventService.cs

```diff
@@ -68,6 +68,10 @@ public Task LogProviderUsersEventAsync(IEnumerable<(ProviderUser, EventType, Dat
         return Task.FromResult(0);
     }
 
+    public async Task LogProviderOrganizationEventsAsync(Guid providerId, IEnumerable<ProviderOrganization> providerOrganizations, EventType type,
```

Comment:
See all the warnings below:

```
This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
```

Looking at the other methods here, you can remove the `async` keyword even though it still returns a `Task`.

Reply:
This was an oversight on my part, I've also set it to return `Task.FromResult(0)` like all the other methods.